qemu/tests/cfg/dump_guest_memory.cfg

```diff
@@ -0,0 +1,52 @@
+- dump_guest_memory:
+    type = dump_guest_memory
+    virt_test_type = qemu
+    no Windows
+    monitors = 'qmp1'
+    monitor_type_qmp1 = qmp
+    x86_64:
+        extra_params = "-device vmcoreinfo"
+    variants:
+        - with_detach_params:
+            cmd_result_check = contain
+            query_qmp_cmd = "query-dump"
+            query_cmd_return_value = "{'status': 'completed'}"
+            qmp_cmd = "dump-guest-memory detach=true, paging=false, protocol=file:/home/dump"
```

Comment:
`/home/dump` should be an individual parameter since you will check its content later.